src/arfsdao.ts

```diff
@@ -636,3 +723,70 @@ export class ArFSPublicDriveBuilder {
 		throw new Error('Invalid drive state');
 	}
 }
+
+export class ArFSPublicFolder extends ArFSEntity implements ArFSFileFolderEntity {
+	constructor(
+		readonly appName: string,
+		readonly appVersion: string,
+		readonly arFS: string,
+		readonly contentType: string,
+		readonly driveId: string,
+		readonly entityType: string,
+		readonly name: string,
+		readonly txId: string,
+		readonly unixTime: number,
+		readonly parentFolderId: string,
+		readonly entityId: string,
```

Comment:
We should use `folderId` here to remain consistent with whats on GQL. Feel free to make a new base type rather than `ArFSFileFolderEntity` to avoid type conflicts

Reply:
Attribute renamed at 2bb63cc8bc91adae85c81a126814ba110a21d489